### ISSUE TITLE
Copy path in JS object (.js) doesn't work

### ISSUE THREAD

**Issue by sanjarcode** [nidu/vscode-copy-json-path#19]
## What's happening
No context option (on right click)

Some examples I tried:
```js
const person = {
  name: "Alice",
  age: 20,
};
```

```js
console.log({
  name: "Alice",
  age: 20,
});

```

**Comment by nidu** [comment]
Hey, sorry, haven't noticed that issue before. If I remember correctly it was done deliberately to not clutter JS context menu since there's already plenty of stuff. Copying with Ctrl+Shift+P should still work there.

**Comment by SimonKib** [comment]
@nidu I don't think this is working

I tried with 

```
// alice.ts
const person: IPerson = {
  name: 'Alice',
  age: 20
}
```
And the toast says "Invalid JSON"

**Comment by nidu** [comment]
Yeah, the extension is parsing JS under the good, so it stops on `: IPerson` as it's invalid JS. Need to use TS parser instead
